docs/swagger/v1.json

```diff
@@ -1,710 +1,690 @@
+          "content": {
+            "application/json": {
               "schema": {
-                "type": "string"
+                "$ref": "#/components/schemas/PayloadMessage"
               }
             }
-          ],
-          "requestBody": {
-            "content": {
-              "application/json": {
-                "schema": {
-                  "$ref": "#/components/schemas/PayloadMessage"
-                }
-              }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Success"
+          }
+        }
+      }
+    },
+    "/api/v1/hubs/{hub}/connections/{connectionId}": {
+      "post": {
+        "tags": [
+          "HubProxyControllerV1"
+        ],
+        "summary": "Send message to the specific connection.",
+        "operationId": "HubProxyControllerV1_POST_SendToConnection",
+        "parameters": [
+          {
+            "name": "hub",
+            "in": "path",
+            "description": "Target hub name, which should start with alphabetic characters and only contain alpha-numeric characters or underscore.",
+            "required": true,
+            "schema": {
+              "type": "string"
             }
           },
-          "responses": {
-            "202": {
-              "description": "Accepted"
+          {
+            "name": "connectionId",
+            "in": "path",
+            "description": "The connection Id.",
+            "required": true,
+            "schema": {
+              "type": "string"
             }
           }
-        }
-      },
-      "/api/v1/hubs/{hub}/connections/{connectionId}": {
-        "post": {
-          "description": "Send message to a single connection Id.",
-          "tags": [],
-          "parameters": [
-            {
-              "name": "hub",
-              "in": "path",
-              "required": true,
-              "description": "Target hub name, which should start with alphabetic characters and only contain alpha-numeric characters or underscore.",
-              "schema": {
-                "type": "string"
-              }
-            },
-            {
-              "name": "connectionId",
-              "in": "path",
-              "required": true,
-              "description": "Target connection Id.",
+        ],
+        "requestBody": {
+          "description": "The message body.",
+          "content": {
+            "application/json": {
               "schema": {
-                "type": "string"
+                "$ref": "#/components/schemas/PayloadMessage"
               }
             }
-          ],
-          "requestBody": {
-            "content": {
-              "application/json": {
-                "schema": {
-                  "$ref": "#/components/schemas/PayloadMessage"
-                }
-              }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Success"
+          }
+        }
+      },
+      "get": {
```

Comment:
not released...

Reply:
will on-hold the change until the new rt is deployed